### DISCUSSION TITLE
Max Players

### DISCUSSION THREAD

**Discussion by klrmac** [LsHallo/ets2-dedicated-convoy-server#5] [Q&A] [answered]
Is it possible to increase the max players from 8? I see some servers in the browser list that show 100+ players available to join the server. How do I set this?

**Comment by LsHallo** [comment]
I don't think it is possible using the vanilla server.
Somebody told me he edited some bytes to allow more players. I did not have time to look into this. If this is feasible to automate (via startup script) we could integrate the functionality into this image as well.

**Comment by JanMaleta** [comment]
This might be worth looking into as there is virtually no informations about how to do that in ATS2/ETS2 and it might be really useful.

**Reply by LsHallo** [reply]
Yeah but you gotta edit a dll file and replace some bytes. I don't have the expertise or time right now to look into this.
As I said previously if somebody figures it out and it doesn't involve downloading a pre edited file from some sketchy website we could integrate this into the image via startup script.

**Reply by JanMaleta** [reply]
do you have the instructions somewhere? I'd love to look into that!

**Reply by LsHallo** [reply]
No I don't know how to do it. I found a youtube video once which had a file in the description. In theory you could replace this in the docker container/mount yourself if you trust the author.

**Reply by bitpredator** [reply]
Hi, I read on the scs software forum that it is possible to create multi-instances on Windows (I currently use this system to have multiple servers with 8 players each, using steam cmd). Do you think it is possible to connect these instances to each other to create a single instance for example with 32 players? If possible, could it also be integrated into your project? By doing this I assume that players don't even have to download modified files.

Greetings Bitpredator

**Reply by LsHallo** [reply]
If someone writes a connection layer to stream players into other instances I would be happy to try and integrate that into this image.
I however can't imagine writing this connection layer myself. So somebody else would have to do this. TruckersMP is an indicator that this can work. I don't know how easy it would be.

SCS also has stated that they want to increase the limit at some point: https://youtu.be/EBXfzksG9IQ?t=5153
(English subtitles available)
But the interview is also nearing 3 years old and nothing indicated, as of now, when and by how much they will increase the limit.

**Comment by LsHallo** [comment, marked as answer]
this has been implemented in #26
